x/evm/keeper/grpc_query.go

```diff
@@ -424,12 +428,11 @@ func (q Querier) ERC20Tokens(c context.Context, req *types.ERC20TokensRequest) (
 		return nil, fmt.Errorf("chain %s not found", req.Chain)
 	}
 
-	if !types.IsEVMChain(chain) {
```

Comment:
this is siily. Maybe remove it.

Reply:
needs more discussion, leaving it in for now